website/js/components/Markdown.js

```diff
@@ -12,8 +12,9 @@ const setInnerHTML = html => el => {
 }
 
 const memoMarked = content => {
-  const result = markStore[content]
-  return result || (markStore[content] = marked(content))
+  if (!markStore[content]) markStore[content] = marked(content) 
```

Comment:
Can we change the check to an `Object#hasOwnProperty()` call instead? This is a more robust test that will not fail if the property evaluates to falsey

```suggestion
  if (!markStore.hasOwnProperty(content)) markStore[content] = marked(content) 
```

Reply:
Well the markdown content is never empty so doesn't really matter

Reply:
Fair enough, and there is no other way to put that key in there at the momen